benchmarks/tpch/prepare.py

```diff
@@ -47,7 +48,15 @@ class PrepareBenchmark(PrepareBenchmarkFactory):
         'supplier': 's'
     }
 
+    def get_copy_cmds(self, table):
```

Comment:
This should be private method, should be named _get_copy_cmds

Reply:
i copied it from the htap benchmark. so if that should be the case then lets do that sort of cleanup in a separate PR?

Reply:
Ok